x-pack/plugins/uptime/public/apps/uptime_app.tsx

```diff
@@ -119,16 +120,20 @@ const Application = (props: UptimeAppProps) => {
                   <UptimeSettingsContextProvider {...props}>
                     <UptimeThemeContextProvider darkMode={darkMode}>
                       <UptimeStartupPluginsContextProvider {...startPlugins}>
-                        <div className={APP_WRAPPER_CLASS} data-test-subj="uptimeApp">
-                          <RedirectAppLinks
-                            className={APP_WRAPPER_CLASS}
-                            application={core.application}
-                          >
-                            <UptimeAlertsFlyoutWrapper />
-                            <PageRouter />
-                            <ActionMenu appMountParameters={appMountParameters} />
-                          </RedirectAppLinks>
-                        </div>
+                        <UptimeIndexPatternContextProvider data={startPlugins.data}>
```

Comment:
Introduced a reference to this context as a way of making the filters work in the alert flyout when it's rendered outside of the Uptime App proper.